backend/core/admin.py

```diff
@@ -140,7 +143,7 @@ class EmailDomainAdmin(admin.ModelAdmin):
     def get_queryset(self, request):
         qst = super().get_queryset(request)
         qst = qst.annotate(
-            _user_number=Subquery(
+            user_number=Subquery(
```

Comment:
W0212: Access to a protected member _user_number of a client class (protected-access)

Reply:
The trick here is that we are creating a protected member, but `pylint` thinks we are updating an existing one.

Using a `_` also bring more confidence in the fact that we are not shadowing an existing model field.

I'd consider acceptable to keep this warning, and to have `_user_number` here.

Reply:
The code that produced the warning is not in this line, it is when another function retrieves _user_number:

``` python
    def user_number(self, obj):
        return obj._user_number
```
In which case it makes sense, you're retrieving a protected member of obj from outside of obj. That's also probably the reason annotate was used instead of alias. 
And it would be pointless to use some Django equivalent of `__getattribute__` just to make the warning disappear, as in `obj.__getattribute__("_n_comparisons_per_criteria")`

=> So, I suggest to still remove the "_"